src/utils/contributors.ts

```diff
@@ -22,14 +22,15 @@ export const getPositionIcon = (position: number): string => {
  * Aggregates contributions by users and determines the top five contributors over the past week.
  */
 export const getTopContributors = (contributions: Contribution[]): Contributor[] => {
-  const oneWeekAgo = new Date();
-  oneWeekAgo.setDate(oneWeekAgo.getDate() - 7);
+  // NOTE: Temporarily showing top 5 contributors of all time instead of only past week's.
+  // const oneWeekAgo = new Date();
+  // oneWeekAgo.setDate(oneWeekAgo.getDate() - 7);
 
-  const filteredContributions = contributions.filter(
-    (contribution) => new Date(contribution.created_date) >= oneWeekAgo,
-  );
+  // const filteredContributions = contributions.filter(
+  //   (contribution) => new Date(contribution.created_date) >= oneWeekAgo,
+  // );
 
-  const contributionSumByUser = filteredContributions.reduce((acc: Record<number, Contributor>, contribution) => {
+  const contributionSumByUser = contributions.reduce((acc: Record<number, Contributor>, contribution) => {
```

Comment:
<img width="316" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/d5822109-b980-4228-a641-ea145e8f907e">

@buckyroberts For now, I'm showing top 5 contributors of `all time` instead of `past week` only, (commented out the logic) and when we have enough weekly contributions, then we can start filtering out and showing past week's only.

What do you think of this approach?

Reply:
Sounds good.